examples/tcpblaster/tcpblaster_client.c

```diff
@@ -218,10 +224,13 @@ void tcpblaster_client(void)
               elapsed.tv_nsec = curr.tv_nsec + borrow;
             }
 
+          char timebuff[100];
```

Comment:
Need move to variable declaration section

Reply:
@xiaoxiang781216 will fix